sfr/subchannel/multiple_SCM_assemblies/19assemblies/tests

```diff
@@ -0,0 +1,9 @@
+[Tests]
+  [generate_mesh]
+    type = CheckFiles
+    input = abr_19assemblies.i
+    check_files = abr_19assemblies_in.e
+    cli_args = "--mesh-only"
+    executable_pattern = 'subchannel*|pronghorn*|combined*'
+  []
+[]
```

Comment:
where are the subchannel tests? you can test them only in opt if they time out in devel

Reply:
see "capabilities" you have that in the other test